decimalSuite_test.go

```diff
@@ -55,73 +55,74 @@ func TestFromSuite(t *testing.T) {
 //
 // }
 
-//getInput gets the test file and extracts test using regex, then returns a map object and a list of test names
+// getInput gets the test file and extracts test using regex, then returns a map object and a list of test names
```

Comment:
nit: Documentation comments should normally be full sentences, including full stops at the end. It doesn't really matter for anything private, but it's something to keep in mind for public symbols, since they will end up in godoc output.